test/inputs-dovetail-frequency-test.js

```diff
@@ -125,7 +125,7 @@ describe('dovetail-frequency', () => {
         type: 'postbytes',
         listenerId: 'listener2',
         timestamp: `${current - 40}`,
-        impressions: [{ campaignId: 400, frequency: "1:1" }],
+        impressions: [{ campaignId: 400, frequency: "1:1" }, { campaignId: 404 }],
```

Comment:
This is what my tests were missing, which is a record with multiple impressions, but at least one has a freq cap, and at least one does not.